### PR TITLE
feat: Unit tests

### PR DESCRIPTION
**Pull request by Vishruta-Patil**
Added Unit tests

### CODE REVIEW

**Review by zoyatanweer — APPROVED** [review]
- code looks clean 
- good to merge

File: src/reducer/quizReducer.test.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+describe("testing quiz", () => {
+  it("get the category of quiz", () => {
+    const action: QuizAction = {
+      type: GET_CATEGORY,
+      payload: "sports",
+    };
+
+    const expectedState = {
+      category: "sports",
+      questionNo: 1,
```
Comment (rkapoor10) [inline src/reducer/quizReducer.test.ts:22]: arrange, act assert is well implemented. Looks good to merge

**Review by rkapoor10 — APPROVED** [review]

File: src/reducer/quizReducer.test.ts [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  GET_RESULT,
+  RESET_CATEGORY_GAME,
+  RESET_DATA,
+  SET_ANSWER_LIST,
+  SET_QUESTION_NUMBER,
+} from "./constant";
+import { quizReducer } from "./quizReducer";
+import { QuizAction } from "./quizReducer.types";
+
+describe("testing quiz", () => {
```
Comment (neetukumari4858) [inline src/reducer/quizReducer.test.ts:13]: I liked that you have used "describe"  for writing multiple tests

**Review by neetukumari4858 — COMMENTED** [review]
- code is readable
- used "Describe" for writingg multiple test that's good.
- good to merge

**Review by neetukumari4858 — APPROVED** [review]

**Pull request merged**